providers/execute.rb

```diff
@@ -5,11 +5,16 @@
 # Author:: Jamie Winsor (<jamie@vialstudios.com>)
 #
 
+require 'etc'
+
 include Chef::Mixin::Rbenv
 
 def load_current_resource
+  @user                         = new_resource.user || Etc.getlogin
+  @home                         = Etc.getpwnam(@user).dir
   @path                         = [ rbenv_shims_path, rbenv_bin_path ] + new_resource.path + system_path
   @environment                  = new_resource.environment
+  @environment["HOME"]          = @home
```

Comment:
For the sake of completeness:

This is patching the rbenv cookbook to add some additional setting of the home and user env vars, which would change behaviour in potentially many places?

Isn't it safer to just ping the version of bundler to the last one which had the previous behaviour which didn't break our builds?